src/schemathesis/__init__.py

```diff
@@ -1,11 +1,3 @@
-from importlib_metadata import PackageNotFoundError, version
-
 from ._hypothesis import register_string_format
 from .loaders import Parametrizer, from_dict, from_file, from_path, from_pytest_fixture, from_uri
 from .models import Case
-
```

Comment:
To be compliant with PEP-0396 we need to keep `__version__` in this module - it could be imported from `constants.py`